jxl/src/frame.rs

```diff
@@ -188,3 +191,39 @@ impl Frame {
         Ok(())
     }
 }
+
+#[cfg(test)]
+mod test_frame {
+    use crate::{
+        bit_reader::BitReader,
+        container::ContainerParser,
+        error::Error,
+        headers::{FileHeader, JxlHeader},
+        util::test::assert_almost_eq,
+    };
+
+    use super::Frame;
+
+    fn read_frame(image: &[u8]) -> Result<Frame, Error> {
```

Comment:
not now, but at some point we might want to move this into src/util/test.rs or similar.